src/container/acrManager.ts

```diff
@@ -226,7 +226,7 @@ export class AcrManager {
                     const username: string = creds.username;
                     const password: string = creds.passwords[0].value;
 
-                    Executor.runInTerminal(`docker login -u ${username} -p ${password} ${registryUrl}`);
+                    Executor.runInTerminal(`iotedgectl login --address ${registryUrl} --username ${username} --password ${password}`);
```

Comment:
On Linux/macOS, it needs to run as sudo

Reply:
Another concern is that currently all `iotedgectl` logic in controlled in IoT Toolkit extension. After this change, IoT Edge extension would also depend on iotedgectl. When iotedgectl is replace with edgeLet, we need to update both two extensions.